lawnchair/src/app/lawnchair/smartspace/provider/SmartspaceWidgetReader.kt

```diff
@@ -57,9 +57,9 @@ class SmartspaceWidgetReader(context: Context) : SmartspaceDataSource(
     }
 
     private fun extractWidgetLayout(appWidgetHostView: ViewGroup): List<SmartspaceTarget> {
-        val children = appWidgetHostView.recursiveChildren.filter { it.isVisible }
+        val children = appWidgetHostView.recursiveChildren
         val texts = children.filterIsInstance<TextView>().filter { !it.text.isNullOrEmpty() }.toList()
-        val images = children.filterIsInstance<ImageView>().toList()
+        val images = children.filterIsInstance<ImageView>().toList().filter { it.drawable != null && it.drawable is BitmapDrawable }
```

Comment:
`t.drawable is BitmapDrawable` should be enough.

Reply:
Already tried and isnt working maybe due to layers in new google at glance widget layout